idunn/utils/geometry.py

```diff
@@ -55,9 +62,21 @@ def parse_poly(lines):
     return MultiPolygon(coords)
 
 
+# Load shape for France
 with open(france_poly_filename) as france_file:
     france_polygon = parse_poly(france_file.readlines())
 
+# Load shape for some cities surrounding
+city_surrounds_polygons = dict()
+
+for filename in os.listdir(cities_surrounds_path):
```

Comment:
An alternative would be to use in a single geojson file with a feature for each city… It would simplify this file handling and need less config, but not sure it would be more flexible. What do you think?

Reply:
I agree that a single file may be easier to handle.  
The config is still useful to be able to allow/disallow some cities on deploy, though.